lib/core/Core.ts

```diff
@@ -75,7 +75,7 @@ export default class Core {
       config.observingIntervalInSeconds
     );
 
-    this.monitor = new Monitor();
+    this.monitor = new Monitor(config);
```

Comment:
Design: similar to earlier comment. This class is relatively "new", thus took the pattern of param-less constructor.